manifests/install.pp

```diff
@@ -0,0 +1,52 @@
+# == Class: nodejs::install
+class nodejs::install inherits nodejs {
```

Comment:
I don't like that this is inheriting nodejs. It's fair to assume that `class { 'nodejs': }` will be declared before `nodejs::install`